### PR TITLE
first commit - html and css built

### PR DESCRIPTION
**Pull request by Sergei-Nikolaienko**
https://Sergei-Nikolaienko.github.io/layout_search-bar-airbnb/
https://Sergei-Nikolaienko.github.io/layout_search-bar-airbnb/report/html_report/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -11,6 +11,29 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      class="upper__form" 
+      action="#" 
+      method="get">
```
Comment (alexandra-protyanova) [inline src/index.html:17]:
```suggestion
    <form 
      class="upper__form" 
      action="#" 
      method="get"
    >
```

Comment (Sergei-Nikolaienko) [reply src/index.html:17]: Apologies, I just noticed the figma prototype being available - will fix shortly.

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  background-size: 11px;
+}
+
+.search__small:focus {
+  background: url(./images/Search.svg) no-repeat 13px center, linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  background-size: 11px;
+}
+
+input::placeholder {
+  font-family: Avenir, sans-serif;
```
Comment (SanyaBratashchuk) [inline src/style.css:70]:
```suggestion
  font-family: inherit;
```

**Review by SanyaBratashchuk — APPROVED** [review]
It looks similiar to mockup, but in future be more careful with data attributes

File: src/style.css (hunk 2 above)
Comment (Sergei-Nikolaienko) [reply src/style.css:70]: Got it, thanks!

**Pull request closed without merging**